InternshipApp/InternshipApp/FlightsApp/ViewModel/FlightViewModel.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  FlightViewModel.swift
+//  InternshipApp
+//
+//  Created by Sead Mašetić on 29. 8. 2024..
+//
+
+import Foundation
+
+@MainActor
+final class FlightViewModel: ObservableObject {
+    @Published var selectedOption: String = "One way"
+    @Published var departureDate: Date = Date()
+    @Published var arrivalDate: Date = Date()
+    @Published var selectedAdults: String = "0"
+    @Published var selectedChildren: String = "0"
+    @Published var selectedClass: String = "Economy"
+    @Published var selectedFlightOption: FlightOption = .arrival
+    
+    @Published var flights: [FlightModel] = [
+        FlightModel(town: "Mostar", airportCode: "MST", airportFullName: "Mostar International Airport", possibleAirports: ["LGA", "IST"]),
+        FlightModel(town: "Sarajevo", airportCode: "SJJ", airportFullName: "Sarajevo International Airport", possibleAirports: ["LGA", "IST"]),
+        FlightModel(town: "New York", airportCode: "LGA", airportFullName: "LaGuardia Airport", possibleAirports: ["SJJ", "MST"]),
+        FlightModel(town: "Istanbul", airportCode: "IST", airportFullName: "Istanbul Airport", possibleAirports: ["SJJ", "MST"])
```

Comment:
Add logic here that each flight has its own available destinations, in case I select Mostar, I can only fly to lets say Sarajevo or Munchen - on the other hand if I select Munchen I can fly to much more locations. Try to add that and filter them so the popover only displays possible destinations